### ISSUE TITLE
A small typo in `graph_properties.cc`

### ISSUE THREAD

**Issue by AmosChenYQ** [tensorflow/tensorflow#56618]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Others

### Source

source

### Tensorflow Version

tf2.10.0

### Custom Code

Yes

### OS Platform and Distribution

Linux Ubuntu 20.04

### Python version

3.8

### Current Behaviour?

```shell
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/grappler/costs/graph_properties.cc#L2145

conflics => conflicts
```

### Standalone code to reproduce the issue

```shell
I think any simple code will output this typo...
```

### Relevant log output

_No response_</details>

**Comment by sushreebarsa** [comment]
@AmosChenYQ Thank you for reporting the issue.
I have raised a PR internally, to fix this one. This ticket will be closed once the PR is merged.
Thank you!

**Comment by mihaimaruseac** [comment]
Closed by #57190
